index.js

```diff
@@ -286,5 +286,24 @@ module.exports = {
 
     self.apos.define('apostrophe-cursor', require('./lib/cursor.js'));
 
+    self.on('apostrophe-pages:notFound', 'softRedirectWithPersona', async (req) => {
+      const soft = self.apos.modules['apostrophe-soft-redirects'];
+      const doc = await self.apos.docs.find(req, { historicUrls: { $in: personify(req.url) } }).sort({ updatedAt: -1 }).toObject();
+      if (!doc) {
```

Comment:
What's the benefit of having 2 `if`s?
Is this working the same way?

```js
     if (!doc || !doc._url) {
        return;
      }
```

Reply:
Either would be fine but in the interest of time since the code is correct I'm hoping we can approve as-is.